CHANGES.md

```diff
@@ -1,3 +1,23 @@
+## edge-19.1.4
+
+* Controller
+  * Added timeouts support! Configurable in the service profiles for each route
```

Comment:
Can we reword this as either "Added timeout support!" or "Added support for timeouts!"?

Reply:
sure